src/js/select.js

```diff
@@ -35,6 +35,24 @@ function initialize(selectEl) {
 
 
 /**
+ * Calculates number of options from a mixed list of options and optgroups.
+ * @param optList - List of option/optgroup.
+ */
+function calculateOptions(optList) {
+  var numOptions = 0;
+
+  Array.from(optList).forEach(function(el) {
```

Comment:
I agree, but since the minified version optimized anyway, sparing a few bytes in the development version for readability should be preferred. But I'll remove it if that is the convention.